### ISSUE TITLE
Helm's YAML warnings differ from kubectl's

### ISSUE THREAD

**Issue by Quentin-M** [kubernetes/helm#1035]
When trying to install (or delete) a chart that generates invalid manifest, the latest _helm_ version reports the following error:
```
Error: release dunking-saola failed: unable to decode "": [pos 292]: only encoded map or array can be decoded into a slice (6)`
```

Creating manually the generated manifest and using _kubectl_ directly to create the resources, we can read a more useful error message:
```
error validating "manifest.yaml": error validating data: [expected type int, for field spec.replicas, got string, found invalid field port for v1.Probe, expected type array, for field spec.template.spec.containers[0].command, got string]; if you choose to ignore these errors, turn validation off with --validate=false
```

Here is my Kubernetes/kubectl versions:
```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"3+", GitVersion:"v1.3.0-alpha.5", GitCommit:"914163247d9a16b46921e83f7dbedb572859b3e4", GitTreeState:"clean", BuildDate:"2016-06-02T20:44:18Z", GoVersion:"go1.6.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"3", GitVersion:"v1.3.0+coreos.0", GitCommit:"1c7c17f64b669d94307f6e8bb09e4ca69f63d6b9", GitTreeState:"clean", BuildDate:"2016-07-02T06:20:28Z", GoVersion:"go1.6.2", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by technosophos** [comment]
It's not so much "inaccurate error reporting" as getting a different message back from the Kube API server. `kubectl` is validating the YAML client-side with its own validation library.

**Comment by philips** [comment]
@technosophos can we use the same validation library as k8s?

**Comment by technosophos** [comment]
This will be solved by #1050, so marking this closed as a duplicate.
